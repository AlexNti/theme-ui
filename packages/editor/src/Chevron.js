/** @jsx jsx */
import { jsx } from 'theme-ui'

export default ({ size = 16, ...props }) => (
  <svg
    {...props}
    viewBox="0 0 16 16"
    width={size}
    height={size}
    sx={{
      pointerEvents: 'none',
    }}>
    <path
      stroke="currentcolor"
      strokeWidth="2"
      fill="none"
      d="M14 6 L8 12 L2 6"
    />
  </svg>
)
