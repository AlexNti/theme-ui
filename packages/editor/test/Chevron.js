import React from 'react'
import renderer from 'react-test-renderer'
import { matchers } from 'jest-emotion'
import Chevron from '../src/Chevron'

test('renders', () => {
  const json = renderer.create(
    <Chevron />
  ).toJSON()
  expect(json).toMatchSnapshot()
})

