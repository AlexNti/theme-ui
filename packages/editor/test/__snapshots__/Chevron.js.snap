// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.emotion-0 {
  pointer-events: none;
}

<svg
  className="emotion-0"
  height={12}
  viewBox="0 0 16 16"
  width={12}
>
  <path
    d="M14 6 L8 12 L2 6"
    fill="none"
    stroke="currentcolor"
    strokeWidth="2"
  />
</svg>
`;
